boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -10,6 +10,39 @@ Zephyr applications use the 96b_carbon configuration to run on the 96Boards
 Carbon hardware. It is based on the STMicroelectronics STM32F401RET Cortex-M4
 CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity.
 
+.. note::
```

Comment:
I've got one nitpick about this:

> The 96b_carbon board is built with two chips

This is misleading, since "96b_carbon" is the Zephyr "BOARD" for the STM32, not the board itself. I think it'd be better to say "96Boards Carbon" to emphasize the physical object.